.github/workflows/smoke-test-prod.yml

```diff
@@ -23,55 +23,55 @@ jobs:
             CARGO_HOME: ~/.cargo
 
     steps:
-    - uses: actions/checkout@v1
-      with:
-        fetch-depth: 3
-    - name: install rust stable
-      uses: actions-rs/toolchain@v1
-      with:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 3
```

Comment:
Please remove the fetch depth in each because v2:
> Fetches only a single commit by default

Reply:
If this has not been changed for some reason, I will remove this.

Reply:
I am sorry, but perhaps I was unclear. I meant the default in v2 is better and preferred and we do not need to specify a fetch-depth anymore. We only need `- uses: actions/checkout@v2`.

Reply:
@imbsky - are you still tracking this request from @jbolda ? I'd like to merge before tomorrows deluge of renovate PRs.

Reply:
@nothingismagick I'm sorry! I was busy with moving and I received a lot of notifications on GitHub recently, so my response was late.

Reply:
No worries! Thanks for finishing it off! You rock ❤️

Reply:
🤘

Reply:
🙌